### PR TITLE
Added editorconfig file

### PR DESCRIPTION
**Pull request by nlok5923**
fixes : #36

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[*.{kt,kts}]
+indent_size = 4
+max_line_length=off
+
+[*.dart]
+indent_brace_style = K&R
+
+[*.md]
+max_line_length = off
+trim_trailing_whitespace = false
```
Comment (aashutoshrathi) [inline .editorconfig:19]: Why false?

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```
Comment (aashutoshrathi) [inline .editorconfig:6]: Are we sure about adding this in `*`

**Review by aashutoshrathi — CHANGES_REQUESTED** [review]
Just verify the comments and also @thecodepapaya what are best practices we follow with Dart

File: .editorconfig (hunk 1 above)
Comment (nlok5923) [reply .editorconfig:19]: Sir i thought newline/whitespace will be preferred in .md files that's why is set `trim_trailing_whitespace = false`  .  but sir if you want i can set it to true .

Comment (aashutoshrathi) [reply .editorconfig:19]: Look for best practices and do that.

Comment (nlok5923) [reply .editorconfig:19]: @aashutoshrathi okay sir i am looking into it.

Comment (nlok5923) [reply .editorconfig:21]: @aashutoshrathi i had a venture through some of the project i had modified editorconfig accordingly.

**Comment by nlok5923** [comment]
@iiitv/project-maintainers sir please review this pull request.

**Review by thecodepapaya — APPROVED** [review]
LGTM

**Comment by nlok5923** [comment]
@aashutoshrathi  sir please review this pull request.

**Review by aashutoshrathi — APPROVED** [review]

**Pull request merged**